snap/snapcraft.yaml

```diff
@@ -104,14 +91,32 @@ slots:
   wayland:
 
 layout:
-  /usr/share:
-    bind: $SNAP/usr/share
+  /usr/share/X11:
+    bind: $SNAP/usr/share/X11
+  /etc/fonts:
+    bind: $SNAP/etc/fonts
+  /usr/share/fonts:
+    bind: $SNAP/usr/share/fonts
+  /usr/share/icons:
+    bind: $SNAP/usr/share/icons
+  /usr/share/glib-2.0/schemas:
+    bind: $SNAP/usr/share/glib-2.0/schemas
+  /usr/lib/$SNAPCRAFT_ARCH_TRIPLET/gdk-pixbuf-2.0:
+    bind: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/gdk-pixbuf-2.0
+  /usr/share/glmark2:
+    bind: $SNAP/usr/share/glmark2
+  /usr/share/libdrm:  # Needed by mesa-core20 on AMD GPUs
+    bind: $SNAP/graphics/libdrm
+  /usr/share/drirc.d:  # Used by mesa-core20 for app specific workarounds
+    bind: $SNAP/graphics/drirc.d
```

Comment:
None of these can be made into environment variables?

Reply:
Fixed